Playground/RxSwiftExtPlayground.playground/Pages/UIScrollView.reachedBottom.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,53 @@
+/*:
+ > # IMPORTANT: To use `RxSwiftExtPlayground.playground`, please:
+
+ 1. Make sure you have [Carthage](https://github.com/Carthage/Carthage) installed
+ 1. Fetch Carthage dependencies from shell: `carthage bootstrap --platform ios`
+ 1. Build scheme `RxSwiftExtPlayground` scheme for a simulator target
+ 1. Choose `View > Show Debug Area`
+ */
+
+//: [Previous](@previous)
+
+import RxSwift
+import RxCocoa
+import RxSwiftExt
+import PlaygroundSupport
+import UIKit
+
+/*:
+ ## reachedBottom
+
+ The `reachedBottom` operator provides a sequence that emits every time scrollview is scrolled to the bottom.
```

Comment:
This should be either
```suggestion
 The `reachedBottom` operator provides a sequence that emits every time the scroll view is scrolled to the bottom.
```

Or: 

```suggestion
 The `reachedBottom` operator provides a sequence that emits every time the `UIScrollView` is scrolled to the bottom.
```